DecipherNow/gm-fabric-dashboard
# Feat: Threads error

Resolves #1268

## Proposed Changes

  - Added threadsError to state
  - Added NotFoundError to ThreadsGrid
  - Moved ThreadsGrid to named file
  
## How to Test

  - Set threads to an empty array, should display "No Threads Found"
  - Run without fabric server and navigate to Threads tab, should display "Failed to Fetch Threads"
